cvmfs/catalog.h

```diff
@@ -231,6 +231,7 @@ class Catalog : public SingleCopy {
   Counters& GetCounters() { return counters_; }
 
   inline const CatalogDatabase &database() const { return *database_; }
+  inline       CatalogDatabase &database()       { return *database_; }
```

Comment:
Let me think a little bit about this and let's discuss this after the DPHEP workshop.